public/app/core/components/AppChrome/ReturnToPrevious/ReturnToPrevious.tsx

```diff
@@ -10,15 +10,15 @@ import { t } from 'app/core/internationalization';
 import { DismissableButton } from './DismissableButton';
 
 export interface ReturnToPreviousProps {
-  href: string;
   title: string;
+  href?: string;
 }
```

Comment:
Why is href now optional for the ReturnToPrevious button? I don't think that makes sense - the button needs an href to do anything, so it should be required.

If _elsewhere_ href is optional, it should be optional there without that bleeding into this component.